src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/Runners/MonitorCollectRunner.CollectionRule.cs

```diff
@@ -4,13 +4,14 @@
 
 using Microsoft.Diagnostics.Monitoring.TestCommon;
 using Microsoft.Diagnostics.Monitoring.TestCommon.Runners;
-using Microsoft.Diagnostics.Tools.Monitor;
 using System;
 using System.Collections.Concurrent;
 using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
 
+using LoggingEventIds = Microsoft.Diagnostics.Tools.Monitor.LoggingEventIds;
```

Comment:
I think you revert these files where you've remove the `using Microsoft.Diagnostics.Tools.Monitor;` lines and added the `using LoggingEventIds = Microsoft.Diagnostics.Tools.Monitor.LoggingEventIds;` lines.